internal/flags.go

```diff
@@ -64,6 +57,8 @@ type cfg struct {
 	ServerCert            string       `json:"serverCert" descr:"A PEM encoded certificate \"file\" for secure artifact download"`
 	DownloadRetryCount    int          `json:"downloadRetryCount" def:"0" descr:"Number of retries, in case of a failed download.\n By default no retries are supported."`
 	DownloadRetryInterval durationTime `json:"downloadRetryInterval" def:"5s" descr:"Interval between retries, in case of a failed download.\n Should be a sequence of decimal numbers, each with optional fraction and a unit suffix, such as '300ms', '1.5h', '10m30s', etc. Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."`
+	InstallPath           string       `json:"installPath" def:"" descr:"A set ot locations on local file system, where to search for module artifacts.\nCommon path separators are supported - \",\", and \"%s\".\nIf empty - the current process directory will be used."`
```

Comment:
Do we need to ensure compatibility with M1 i.e. to ensure that M1 updates are supported by M2?

Reply:
Checked, it is M1 compatible